lms/templates/instructor/instructor_dashboard_2/send_email.html

```diff
@@ -98,28 +95,18 @@
 
     <hr>
 
-    <div class="vert-left email-background" id="section-task-history">
-        <h2> ${_("Email Task History")} </h2>
-        <div>
-            <p>${_("To see the content of previously sent emails, click this button:")}</p>
-            <br/>
-            <input type="button" name="task-history-email-content" value="${_("Show Sent Email History")}" data-endpoint="${ section_data['email_content_history_url'] }" >
-            <div class="content-request-response-error msg msg-warning copy"></div>
-            <p>
-            <div class="content-history-email-table">
-                <p><em>${_("To read a sent email message, click its subject.")}</em></p>
-                <br/>
-                <div class="content-history-table-inner"></div>
-            </div>
-            <div class="email-messages-wrapper"></div>
-        </div>
-        <div>
-            <p>${_("To see the status for all email tasks submitted for this course, click this button:")}</p>
```

Comment:
This should still exist (unless @clrux intended to remove it), it's in master: https://github.com/edx/edx-platform/pull/12602#discussion_r65573428